src/test/java/org/polyjdbc/core/integration/TestSchemaManager.java

```diff
@@ -25,12 +25,19 @@
 
     private Schema schema;
 
-    public TestSchemaManager(PolyJDBC polyJDBC) {
+    private String schemaName;
+
+    public TestSchemaManager(PolyJDBC polyJDBC, String schemaName) {
         this.polyJDBC = polyJDBC;
+        this.schemaName = schemaName;
     }
 
     public void createSchema() {
-        schema = new Schema(polyJDBC.dialect());
+        if (schemaName == null) {
```

Comment:
you don't need this if -
`new Schema(polyJDBC.dialect(), schemaName)` is happy with null schemaName